python/ray/ml/train/integrations/rl/rl_trainer.py

```diff
@@ -0,0 +1,201 @@
+from typing import Optional, Dict, Type, Union, Callable, Any
+
+from ray.ml.checkpoint import Checkpoint
+from ray.ml.config import ScalingConfig, RunConfig
+from ray.ml.preprocessor import Preprocessor
+from ray.ml.trainer import Trainer, GenDataset
+from ray.rllib.agents.trainer import Trainer as RLLibTrainer
+from ray.rllib.utils.typing import PartialTrainerConfigDict, EnvType
+from ray.tune import Trainable, PlacementGroupFactory
+from ray.tune.logger import Logger
+from ray.tune.registry import get_trainable_cls
+from ray.tune.resources import Resources
+from ray.util.annotations import PublicAPI
+from ray.util.ml_utils.dict import merge_dicts
+
+
+@PublicAPI(stability="alpha")
+class RLTrainer(Trainer):
+    """Reinforcement learning trainer.
+
+    This trainer provides an interface to RLLib trainables.
+
+    If datasets and preprocessors are used, they can be utilized for
+    offline training, e.g. using behavior cloning. Otherwise, this trainer
+    will use online training.
+
+    Args:
+        algorithm: Algorithm to train on. Can be a string reference,
+            (e.g. ``"PPO"``) or a RLLib trainer class.
+        scaling_config: Configuration for distributed training, e.g. number
+            of workers or resources per worker.
+        run_config: Run config passed to ``Tuner()``
```

Comment:
Do we want users to be aware that ``Tuner`` is being used? It seems like an implementation detail. Can we just use the same docstring for `run_config` that we have in the other Trainers?

Reply:
RunConfig is not a Tuner specific concept right?

Reply:
Copied the docstrings from the basetrainer